packages/sanity/src/desk/panes/document/documentPanel/documentViews/FormView.tsx

```diff
@@ -152,6 +155,14 @@ export const FormView = forwardRef<HTMLDivElement, FormViewProps>(function FormV
     >
       <PresenceOverlay margins={margins}>
         <Box as="form" onSubmit={preventDefault} ref={setRef}>
+          {/* This is the document title */}
+          <Box marginBottom={5}>
+            <ElementQuery>
+              <Title forwardedAs="h2" muted={!title}>
```

Comment:
Changed from `forwardedAs="h6"`, as this title is now much more prominent (but not marked as `h1` as there can be multiple instances of these at any one time)